concepticondata/conceptlists/Bowern-2021-203c.tsv

```diff
@@ -0,0 +1,204 @@
+ID	NUMBER	ENGLISH	DATABASE_ID	SEMANTIC_CATEGORY	LIST	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Bowern-2021-203c-1	1	acorn bread	644	food	Culture Vocabulary		
+Bowern-2021-203c-2	2	acorn mush 	601	food	Culture Vocabulary		
+Bowern-2021-203c-3	3	afterworld, land of dead, Heaven 	603	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-4	4	alcohol 	652	food	Culture Vocabulary	404	ALCOHOL (FERMENTED DRINK)
+Bowern-2021-203c-5	5	armband 	764	dress	Culture Vocabulary		
+Bowern-2021-203c-6	6	arrow 	605	subsistence tool	Culture Vocabulary	977	ARROW
+Bowern-2021-203c-7	7	arrow straightener 	606	other	Culture Vocabulary		
+Bowern-2021-203c-8	8	awl 	765	subsistence tool	Culture Vocabulary	278	AWL
+Bowern-2021-203c-9	9	axe/stone axe 	707	subsistence tool	Culture Vocabulary	677	AXE
+Bowern-2021-203c-10	10	bag (string) 	766	culture-material	Culture Vocabulary		
+Bowern-2021-203c-11	11	bag, sack 	684	culture-material	Culture Vocabulary	2468	BAG
+Bowern-2021-203c-12	12	bait for fishing 	607	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-13	13	ball game 	602	food	Culture Vocabulary		
+Bowern-2021-203c-14	14	basin/bowl 	708	culture-material	Culture Vocabulary	915	SINK (BASIN)
+Bowern-2021-203c-15	15	basket (general) 	610	culture-material	Culture Vocabulary	1539	BASKET
+Bowern-2021-203c-16	16	basket, small 	611	culture-material	Culture Vocabulary		
+Bowern-2021-203c-17	17	basketry hat 	612	culture-material	Culture Vocabulary		
+Bowern-2021-203c-18	18	bead 	613	culture-material	Culture Vocabulary	1860	BEAD
+Bowern-2021-203c-19	19	beans 	701	food	Culture Vocabulary	832	BEAN
+Bowern-2021-203c-20	20	bed 	711	culture-material	Culture Vocabulary	1663	BED
+Bowern-2021-203c-21	21	bench, seat 	712	culture-material	Culture Vocabulary		
+Bowern-2021-203c-22	22	blowgun 	713	subsistence tool	Culture Vocabulary	432	BLOWGUN
+Bowern-2021-203c-23	23	body feather decorations 	767	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-24	24	boomerang (number 7) 	769	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-25	25	boomerang (small) 	770	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-26	26	boomerang/throwing stick (generic) 	768	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-27	27	bottle 	618	acculturation	Culture Vocabulary	1321	BOTTLE
+Bowern-2021-203c-28	28	bottom grinding stone 	375	manufacture	Culture Vocabulary	3023	BOTTOM GRINDSTONE
+Bowern-2021-203c-29	29	bow 	615	subsistence tool	Culture Vocabulary	994	BOW
+Bowern-2021-203c-30	30	bowl (coolamon) 	771	culture-material	Culture Vocabulary	1530	BOWL
```

Comment:
"coolamon" seems to be more like a vessel and in another Bowern list is was mapped to BASKET.

Reply:
In the same list, we already have basket (general), so I would rather unmap this concept.

Reply:
Sounds good!